pkgs/servers/nosql/neo4j/default.nix

```diff
@@ -11,18 +11,24 @@ stdenv.mkDerivation rec {
     sha256 = "1zjb6cgk2lpzx6pq1cs5fh65in6b5ccpl1cgfiglgpjc948mnhzv";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ makeWrapper bashInteractive ];
 
   installPhase = ''
     mkdir -p "$out/share/neo4j"
     cp -R * "$out/share/neo4j"
 
     mkdir -p "$out/bin"
+    compgen_wrapper="$out/share/neo4j/bin/compgen"
```

Comment:
small question.
the mkdir above create `$out/share/neo4j`
is the `bin` subdirectory created as well somehow ?
cat won't create the subdirectory, I think.

Reply:
oh, everything gets copied from the $out in the step above. Nevermind then.